packages/debugger/lib/controller/sagas/index.js

```diff
@@ -92,8 +88,10 @@ function* stepNext() {
     (!upcoming ||
       !upcoming.node ||
       isDeliberatelySkippedNodeType(upcoming.node) ||
-      (upcoming.sourceRange.start == startingRange.start &&
-        upcoming.sourceRange.length == startingRange.length))
+      (upcoming.sourceRange.start === starting.sourceRange.start &&
+        upcoming.sourceRange.length === starting.sourceRange.length &&
+        upcoming.source.id === starting.source.id &&
+        upcoming.source.compilation === starting.source.compilation))
```

Comment:
`upcoming.source.compilation.id`??

Reply:
Well, `upcoming.source.compilationId`, but yes, point taken. :P

Reply:
ah right because `id` here means "index"? I was just commenting on the asymmetry